src/medCore/database/medDatabaseController.cpp

```diff
@@ -587,13 +592,6 @@ bool medDatabaseController::moveDatabase( QString newLocation)
             qDebug() << "Restarting connection...";
             this->createConnection();
         }
-
-        // now delete the old archive
-        if(medStorage::removeDir(oldLocation))
```

Comment:
Won't merge until resolution on this topic.

For me - deleting the old DB should be a prompt, rather than mandated. This permits us to preserve DBs if we so choose. Users won't change DBs frequently enough to be bothered by an extra step?

If this is acceptable, let me know and I'll implement the change myself.